### PR TITLE
get vcforpython27.msi path

### PR DESCRIPTION
**Pull request by rebeldroid12**
specify user path for where vcforpython27.msi is located.

### CODE REVIEW

**Comment by hectron** [comment]
This is pretty straight forward. It'll be helpful for the people who are copy and pasting the commands from the read me. Hopefully they see the `YOUR-USER` section.

**Comment by rebeldroid12** [comment]
the previous version didn't make it clear what was going on. i think this version does :)

File: docs/Windows_Install_Instructions.md [hunk 1]
```diff
@@ -86,9 +86,9 @@ Install python dependencies (while in virtual environment aka (alliance)):
 	requests==2.7.0
 
 
-**If you get an error about psycopg2, go to: http://aka.ms/vcpython27 & download. Run the following command and retry to install the requirements again.
+**If you get an error about psycopg2, go to: http://aka.ms/vcpython27 & download. Run the following command and retry to install the requirements again. (make sure to get the correct path to where VCForPython27.msi is located)
 
-	msiexec/i C:\Users\\Downloads\VCForPython27.msi ALLUSERS=1
+	msiexec/i C:\Users\YOUR-USER\Downloads\VCForPython27.msi ALLUSERS=1
```
Comment (hectron) [inline docs/Windows_Install_Instructions.md]: @rebeldroid12 what you can also do is simply use the following command:

    msiexec /i %USERPROFILE%\Downloads\VCForPython27.msi ALLUSERS=1

I just saw that the `/i` is appended to the `msiexec` command, which might result in an error. Also, the `%USERPROFILE%` variable resolves to the current user's path. E.g. `C:\Users\Beloved`.

How does that change sound?

**Comment by rebeldroid12** [comment]
Good catch!

**Pull request merged**